docs-ref-conceptual/spring-framework/deploy-spring-boot-java-app-with-maven-plugin.md

```diff
@@ -94,35 +95,47 @@ In this section, you will configure the Spring Boot project `pom.xml` so that Ma
 
    ```xml
    <plugin>
-    <groupId>com.microsoft.azure</groupId>
-    <artifactId>azure-webapp-maven-plugin</artifactId>
-    <version>1.4.0</version>
-    <configuration>
-      <deploymentType>jar</deploymentType>
-
-      <!-- configure app to run on port 80, required by App Service -->
-      <appSettings>
-        <property> 
-          <name>JAVA_OPTS</name> 
-          <value>-Dserver.port=80</value> 
-        </property> 
-      </appSettings>
-
-      <!-- Web App information -->
-      <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
-      <appName>${WEBAPP_NAME}</appName>
-      <region>${REGION}</region>  
-
-      <!-- Java Runtime Stack for Web App on Linux-->
-      <linuxRuntime>jre8</linuxRuntime>
-    </configuration>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-webapp-maven-plugin</artifactId>
+      <version>1.6.0</version>
+      <configuration>
+         <schemaVersion>V2</schemaVersion>
+         <subscriptionId>${SUBSCRIPTION_ID}</subscriptionId>
+         <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
+         <appName>${WEBAPP_NAME}</appName>
+         <region>${REGION}</region>
+         <pricingTier>P1V2</pricingTier>
+         <runtime>
+         <os>linux</os>
+         <javaVersion>jre8</javaVersion>
+         <webContainer>jre8</webContainer>
+         </runtime>
+         <appSettings>
+         <!-- Set port forwarding for app service-->
+         <property>
+            <name>PORT</name>
+            <value>8080</value>
```

Comment:
Doesn't App Service for Linux expects services to run on PORT 80?

Reply:
Yes, this is to redirect traffic for port 80 to 8080

Reply:
@brunoborges  Thanks for the information, so spring-boot projects could be deployed to Azure app service directly if the port is not changed, and this part is not necessary here as users could access their app service without this config. So shall we remove this part or made it a comment to help users set the port forwarding if their app services use another port?

Reply:
@brunoborges Thanks for your information, it seems that now we can deploy a spring-boot project to JavaSE runtime directly, so should we remove this app setting? Or keep it as a comment to tell users how to set port forwarding when their app service runs on another port?

Reply:
Just remove it. Instead, provide a note in the docs that for Java SE apps, there will be already an environment variable named PORT that will indicate which port should be used by the app.

Users then have to indicate whatever Java framework they are using, to reference this value.

Reply:
@brunoborges corresponding changes has been committed, please review.